pnpm-workspaces/packages/two/src/main.js

```diff
@@ -0,0 +1,14 @@
+const cowsay = require('cowsay');
+const First = require('@bazel-poc/first');
+
+module.exports = two = () => {
```

Comment:
What's the `two = ` here?

Reply:
I belive is just naming the function, but I can probably return an `anonymous` function instead.